contribution.md

```diff
@@ -0,0 +1,115 @@
+
+    `<topic>/<issue_title_or_description>`
+
+   with `fix`, `feature`, `hotfix`, `release`, `ci`... as possible `<topic>` values.
+
+#### Commit policy:
+
+Here are some good practices to apply:
+
+ - Use English for all contributions
+ - __Write descriptive and meaningful commit messages__. No, "change" "wip" or "update" don't belong to this category!
+ - __Commit only related work__: the least amount of lines that make sense together, and related to the branch purpose.
+ - Don't commit changes that break the code: projects should be compiling at any commit level.
+ - The author of a commit is responsible for it and should try to fix issues it may cause.
+
+#### Push consideration:
+
+ - Rebase your working branch as frequently as you can before publishing it on the remote repository.
+ - Don't push directly on main branches (master, main, develop...)
+ - __Never `push --force` onto a main branch__ (master, main, develop), other developers are basing there work on it!
+ - You may rewrite the history (`push --force`) in specific cases:
+     - After you rebased your branch
+     - To amend the last commit of your branch
+
+    __Always make sure to explicit the target branch when using `push --force`!__
+
+
+Overall, __avoid changing a published history__! If you want to change your branch history:
+ 1. Create a new branch
+ 2. Re-commit your changes properly on the new branch (`git cherry-pick` can be helpful)
+ 3. Then remove the former branch
+
+
+### Gitlab environment
+
+The Gitlab platform is used to centralize the code, and to manage to development of Media-Cloud AI.
+
+Before starting your contribution, please ensure your Gitlab profile shares the email address you configured in your local Git.
+
+#### Issues:
+
+Each contribution must be linked to an issue on the related Gitlab repository.
+
+This means that, as an example, if you found a bug:
+
+ 1. Check whether an issue already exists regarding this bug
+ 2. If necessary, create a new issue to report the bug
+ 3. Create a new branch with a name based on the title of the issue (see the [Branch naming rule](#branch-naming-rule) section)
+ 4. Start commiting patches to fix this bug
+
+#### Merge Requests:
+
+Each contribution must result in a Merge Request on the related Gitlab repository.
+
+When your development is complete, push your branch on the remote Gitlab repository, and create a Merge Request targetting a main branch (`develop` or `main`).
+
+You must __assign yourself as an Assignee__: meaning that you are responsible for the changes proposed, and you will be in charge of merging the branch into the target branch.
+
+You can mark the Merge Request as _Draft_, if you consider it is not ready for a review.
+
+When the Merge Request is ready, you have to request a review by __assigning the issuer or a relevant* developer as a Reviewer__: this person has to review your code, may request some changes, and is responsible for the final approval.
```

Comment:
Maybe it could be worth mentionning that we've got a Q&A team on backend repositories? (what means that the reviewer should be part of the Q&A team)